src/sql/workbench/contrib/query/browser/queryActions.ts

```diff
@@ -224,7 +224,7 @@ export class RunQueryAction extends QueryTaskbarAction {
 		if (!this.editor.isSelectionEmpty()) {
 			if (this.isConnected(this.editor)) {
 				// If we are already connected, run the query
-				this.runQuery(this.editor, true);
+				this.runQuery(this.editor);
```

Comment:
How is this fixing it? The parameter is literally called runCurrentStatement and defaults to false so this seems like the correct value to me. 

The bug reports seem to indicate that this only is happening when you don't have something selected. It's supposed to just run the single line you're on but it looks like that broke somewhere.

Reply:
Sorry about that. I confused this command with the run command. Making a new fix right now